app/main/views/marketplace.py

```diff
@@ -119,6 +116,15 @@ def copyright():
     return render_template('content/copyright.html')
 
 
+@main.route('/terms-of-use')
+def terms_of_use():
+    current_terms_date = current_app.config['DM_CURRENT_TERMS_DATE']
```

Comment:
so the workflow will be create new file, edit template and then change env var?

wouldn't it be easier esp for content editors to determine latest using a version number eg. /1.html ?